hrms/payroll/doctype/payroll_settings/payroll_settings.json

```diff
@@ -107,13 +109,23 @@
    "fieldname": "define_opening_balance_for_earning_and_deductions",
    "fieldtype": "Check",
    "label": "Define Opening Balance for Earning and Deductions"
+  },
+  {
+   "default": "0",
+   "fieldname": "include_zero_value_salary_component_in_salary_slip",
+   "fieldtype": "Check",
+   "label": "Include Zero Value Salary Component in Salary Slip"
+  },
```

Comment:
```suggestion
  {
   "default": "0",
   "fieldname": "show_zero_value_components",
   "fieldtype": "Check",
   "label": "Show Zero Value Components in Salary Slip"
  },
```


- Let's name it "Show Zero Value Components in Salary Slip" to be consistent with the other similar settings like show leave balance? Also removing the "salary" prefix for components since its obvious.
- The fields marked in orange are looking ugly since the descriptions are longer. Instead of a 3-column layout, move related settings to a separate section.
- Let's keep the fieldnames as short as possible to make the code more readable (`show_zero_value_components` instead of `include_zero_value_salary_component_in_salary_slip`? `in_salary_slip` can be omitted since it's obvious in `salary_slip.py`

<img width="1331" alt="component" src="https://user-images.githubusercontent.com/24353136/220550060-420eebe1-30c2-495a-8568-13e924274acb.png">